src/puppetlabs/puppetdb/scf/storage_utils.clj

```diff
@@ -252,3 +253,28 @@ must be supplied as the value to be matched."
        (sql/do-commands
         (str "ALTER TABLE " table " ALTER COLUMN " column
              " RESTART WITH " restartid))))))
+
+(defn clj->pgobject
```

Comment:
technically this isn't a conversion to just a pgobject, its a conversion to specifically a json type pgobject.